cmd/nerdctl/load_encrypt_linux_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/rootlessutil"
+	"github.com/containerd/nerdctl/pkg/testutil"
+	"github.com/containerd/nerdctl/pkg/testutil/testregistry"
+	"gotest.tools/v3/assert"
+)
+
+func TestLoadEncryptImage(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	keyPair := newJWEKeyPair(t)
+	defer keyPair.cleanup()
+
+	// copy temp private key to default directory, from default config.toml --decryption-keys-path value
+	ocicryptDefaultPath := "/etc/containerd/ocicrypt/keys"
+	if rootlessutil.IsRootless() {
+		ocicryptDefaultPath = "~/.config/containerd/ocicrypt/keys"
+	}
+	err := os.MkdirAll(ocicryptDefaultPath, 0755)
+	assert.NilError(t, err)
+	exec.Command("cp", "-f", keyPair.prv, ocicryptDefaultPath)
```

Comment:
Maybe, we should change the file name from `mykey.pem`, `mypubkey.pem` to `nerdctl-TestLoadEncryptImage-key.pem`, `nerdctl-TestLoadEncryptImage-pubkey.pem` to avoid confusion